master/getting-started/openshift/installation.md

```diff
@@ -89,3 +107,11 @@ node1
 [etcd]
 etcd1
 ```
+
+
+## Notes
+
+- Upgrades from existing OpenShift deployments (running openshift-sdn, or the OpenShift build of flannel) to Calico have not been tested.
+
+- These instructions outline deployment on standard RHEL7 or CentOS7 hosts. For deployment on other platforms, please reach out to the Calico team via the [Calico-Users slack channel](https://www.projectcalico.org/community#slack).
```

Comment:
I would suggest "via the [Calico-Users slack channel](https://www.projectcalico.org/community#slack)" --> "on [Slack](https://www.projectcalico.org/community#slack)"